server/py/services/api/utils/notification_pusher.py

```diff
@@ -198,5 +201,22 @@ def resolve_notifications_default_params():
         NotificationTypes.git: {},
         NotificationTypes.ipython: {},
         NotificationTypes.slack: {},
+        NotificationTypes.mail: _get_mail_notification_default_params(),
         NotificationTypes.webhook: {},
     }
+
+
+def _get_mail_notification_default_params():
+    global mail_notification_default_params
+    if mail_notification_default_params is not None:
+        return mail_notification_default_params
```

Comment:
In the future this will be part of the `Notification` class and the `mail_notification_default_params` can live as a static member there.

Thinking if its worth making this function(and `resolve_notifications_default_params`) as part of the server side `NotificationPusher` class above, so this can still be a static member here instead of a global, which can often be confusing.